src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -91,53 +96,60 @@ def __init__(self, startNetworkThread=True):
         self._InitLib()
 
         devCtrl = c_void_p(None)
-        res = self._dmLib.nl_Chip_DeviceController_NewDeviceController(pointer(devCtrl))
+        res = self._dmLib.nl_Chip_DeviceController_NewDeviceController(
+            pointer(devCtrl))
         if res != 0:
             raise self._ChipStack.ErrorToException(res)
-        
+
         pairingDelegate = c_void_p(None)
-        res = self._dmLib.nl_Chip_ScriptDevicePairingDelegate_NewPairingDelegate(pointer(pairingDelegate))
+        res = self._dmLib.nl_Chip_ScriptDevicePairingDelegate_NewPairingDelegate(
+            pointer(pairingDelegate))
         if res != 0:
             raise self._ChipStack.ErrorToException(res)
-        
-        res = self._dmLib.nl_Chip_DeviceController_SetDevicePairingDelegate(devCtrl, pairingDelegate)
+
+        res = self._dmLib.nl_Chip_DeviceController_SetDevicePairingDelegate(
```

Comment:
this reformatting looks extraneous, can you revert these until we have python in restyled?

Reply:
Reverted most style changes, needs a pr to reformat all python scripts.